### ISSUE TITLE
Crash when dropping music files into playlist after cold launch.

### ISSUE THREAD

**Issue by unmec** [iina/iina#2976]
**System and IINA version:**

- macOS Mojave 10.14.6
- IINA 1.0.6 Build 113, mpv 0.31.0

**Expected behavior:**
Should not crash

**Actual behavior:**
Crash when click on shuffle mode

<details>
<summary><b>Crash report:</b></summary>

```
Process:               IINA [52946]
Path:                  /Applications/IINA.app/Contents/MacOS/IINA
Identifier:            com.colliderli.iina
Version:               1.0.6 (113)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           IINA [52946]
User ID:               501

Date/Time:             2020-06-11 *********************
OS Version:            Mac OS X 10.14.6 (18G5033)
Report Version:        12
Anonymous UUID:        *********************

Sleep/Wake UUID:       *********************

Time Awake Since Boot: 190000 seconds
Time Since Wake:       49000 seconds

System Integrity Protection: enabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       EXC_I386_GPFLT
Exception Note:        EXC_CORPSE_NOTIFY

Termination Signal:    Segmentation fault: 11
Termination Reason:    Namespace SIGNAL, Code 0xb
Terminating Process:   exc handler [52946]

Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   com.colliderli.iina           	0x0000000100c9c3c3 specialized Dictionary.subscript.getter + 19
1   com.colliderli.iina           	0x0000000100c917f3 PlaylistViewController.refreshTotalLength() + 771
2   com.colliderli.iina           	0x0000000100c95bc4 PlaylistViewController.tableView(_:viewFor:row:) + 5108
3   com.colliderli.iina           	0x0000000100c96d5d @objc PlaylistViewController.tableView(_:viewFor:row:) + 77
4   com.apple.AppKit              	0x00007fff36f22514 -[NSTableView(NSTableViewViewBased) _delegate_viewForTableColumn:row:] + 88
5   com.apple.AppKit              	0x00007fff36f220ca -[NSTableView(NSTableViewViewBased) makeViewForTableColumn:row:] + 91
6   com.apple.AppKit              	0x00007fff36f21c20 -[NSTableRowData _addViewToRowView:atColumn:row:] + 342
7   com.apple.Foundation          	0x00007fff3ba56d92 __NSIndexSetEnumerate + 706
8   com.apple.AppKit              	0x00007fff37735aa7 __56-[NSTableRowData reloadDataForRowIndexes:columnIndexes:]_block_invoke.1252 + 332
9   com.apple.AppKit              	0x00007fff36ef90f4 -[NSTableRowData enumerateAvailableRowViewsUsingBlock:] + 134
10  com.apple.AppKit              	0x00007fff3718fe0e -[NSTableRowData reloadDataForRowIndexes:columnIndexes:] + 393
11  com.apple.AppKit              	0x00007fff37168565 -[NSTableView reloadDataForRowIndexes:columnIndexes:] + 112
12  com.colliderli.iina           	0x0000000100c96cd7 closure #1 in closure #2 in PlaylistViewController.tableView(_:viewFor:row:) + 343
13  com.colliderli.iina           	0x0000000100cde409 thunk for @escaping @callee_guaranteed () -> () + 25
14  libdispatch.dylib             	0x00007fff656635f8 _dispatch_call_block_and_release + 12
15  libdispatch.dylib             	0x00007fff6566463d _dispatch_client_callout + 8
16  libdispatch.dylib             	0x00007fff6566f68d _dispatch_main_queue_callback_4CF + 1135
17  com.apple.CoreFoundation      	0x00007fff39770967 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
18  com.apple.CoreFoundation      	0x00007fff39770091 __CFRunLoopRun + 2289
19  com.apple.CoreFoundation      	0x00007fff3976f54e CFRunLoopRunSpecific + 455
20  com.apple.HIToolbox           	0x00007fff389ce1ab RunCurrentEventLoopInMode + 292
21  com.apple.HIToolbox           	0x00007fff389cdee5 ReceiveNextEventCommon + 603
22  com.apple.HIToolbox           	0x00007fff389cdc76 _BlockUntilNextEventMatchingListInModeWithFilter + 64
23  com.apple.AppKit              	0x00007fff36d6577d _DPSNextEvent + 1135
24  com.apple.AppKit              	0x00007fff36d6446b -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 1361
25  com.apple.AppKit              	0x00007fff36d5e588 -[NSApplication run] + 699
26  com.apple.AppKit              	0x00007fff36d4dac8 NSApplicationMain + 777
27  com.colliderli.iina           	0x0000000100b1cf39 main + 9
28  libdyld.dylib                 	0x00007fff656b13d5 start + 1
```

</details>

<br>

**Steps to reproduce:**

- [ Unknown ] MPV does not have this problem.
Cold launch IINA by double click a music file, then I drag and drop about 180+ music files(mostly mp3, along with a few flac, m4a, mp4) into playlist, then click on shuffle mode, INNA crashes immediately.

**How often does this happen?**
I can reproduce this crash everytime when adding my 180+ music files, however, INNA seems working fine with some subsets of these music files. Unknown if it's due to a few corrupted files, please note that all files can be correctly played when open individually by INNA though.

**Comment by alejx** [comment]
Shoud be fixed in 1.0.7.

**Comment by alejx** [comment]
Duplicate of #2745

**Comment by unmec** [comment]
Confirmed: it's fixed with 1.0.7 b2
Thanks!
